libs/ui/src/lib/search-hit/index.tsx

```diff
@@ -0,0 +1,55 @@
+import {localization} from '@catalog-frontend/utils';
+import {getTranslateText as translate} from 'libs/utils/src/lib/language/translateText';
+import styles from './search-hit.module.css';
+
+export function SearchHit({searchHit}: any) {
```

Comment:
Det er god praksis å unngå `any` for å utnytte typescript mest mulig, men skjønner at det ikke lar seg gjøre i noen sjeldne tilfeller. En alternativ løsning er å defienere `interface` med typene. 
Eks:  https://github.com/Informasjonsforvaltning/catalog-frontend/pull/46/files#diff-ce53fd4a3b2853e1fa55e55cd6e310671320d09e76c5c89dc587d8672ea7875dR24

Reply:
Wops, glemte det bare, fikset nå!